features/registry/is.feature

```diff
@@ -86,3 +86,87 @@ Feature: Testing imagestream
       | <%= cb.digest2 %> |
 
+  # @author xiuwang@redhat.com
+  # @case_id OCP-19196
+  @destructive
+  @admin
+  Scenario: Prune images when DC reference to invalid image
+    Given I have a project
+    Given I enable image-registry default route
+    Given default image registry route is stored in the :registry_hostname clipboard
+    Given certification for default image registry is saved
+    When I run the :new_app client command with:
+      | app_repo | ruby~https://github.com/sclorg/ruby-ex.git |
+    Then the step should succeed
+    And the "ruby-ex-1" build was created
+    And the "ruby-ex-1" build completes
+    When I run the :patch client command with:
+      | resource      | dc                                              |
+      | resource_name | ruby-ex                                         |
+      | p             | {"spec":{"triggers":[{"type":"ConfigChange"}]}} |
+    Then the step should succeed
+    When I run the :patch client command with:
+      | resource      | dc                                                                                                                                                |
+      | resource_name | ruby-ex                                                                                                                                           |
+      | p             | {"spec":{"template":{"spec":{"containers":[{"image":"<%= cb.registry_hostname %>/<%= project.name %>/ruby-ex@sha256:nonono","name":"ruby-ex"}]}}}}|
+    Given cluster role "system:image-pruner" is added to the "first" user
+    And I run the :oadm_prune_images client command with:
+      | keep_tag_revisions    | 1                           |
+      | keep_younger_than     | 0                           |
+      | registry_url          | <%= cb.registry_hostname %> |
+      | confirm               | true                        |
+      | certificate_authority | <%= cb.reg_crt_name %>      |
+    Then the step should fail
+    And the output should contain:
+      | invalid container image reference |
+    And I run the :oadm_prune_images client command with:
+      | keep_tag_revisions    | 1                           |
+      | keep_younger_than     | 0                           |
+      | registry_url          | <%= cb.registry_hostname %> |
+      | confirm               | true                        |
+      | certificate_authority | <%= cb.reg_crt_name %>      |
+      | ignore_invalid_refs   | true                        |
+    Then the step should succeed
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-16495
+  @destructive
+  @admin
+  Scenario: Do not prune layer of a valid Image due to minimum aging
+    Given I have a project
+    And evaluation of `project.name` is stored in the :u1p1 clipboard
+    Given I enable image-registry default route
+    Given default image registry route is stored in the :registry_hostname clipboard
+    Given certification for default image registry is saved
+
```

Comment:
There are some blank lines, is ti intended?

Reply:
It's intended